src/main/java/com/rackspace/jenkins_nodepool/NodePoolSlave.java

```diff
@@ -321,14 +366,24 @@ private void printBuildDetails(RunList builds) {
         }
     }
 
+    private int getBuildLog(final Run build, final int maxLines) {
```

Comment:
This should be `getBuildLogSize`, though I think its of little value as its going to return `100` in most cases.

Reply:
Removing - was seeing if we could determine anything from this.